README.md

```diff
@@ -2,10 +2,121 @@
 
 Node SDK for the Flow Commerce API.
 
-# Documentation / API
+## Installation
 
-Coming Soon.
+    npm install @flowio/node-sdk
 
-# Development / Release
+## Documentation / API
+
+The official documentation can be found at [docs.flow.io](https://docs.flow.io).
+This will cover in detail all of the resources of the api.
+
+See [API Resources](docs/README.md) for the documentation of the Node SDK API
+and its functions.
+
+### Getting Started
+
+#### Authentication
+
+Visit [console.flow.io](https://console.flow.io) to create an account and
+generate an API key.
+
+#### Making an authenticated request
+
+Most requests to `api.flow.io` require _Basic Authentication_. Here is how to
+make and authenticated request with JavaScript.
+
+```JavaScript
+import client from '@flowio/node-sdk';
+
+const api = client();
+const token = process.env.FLOW_IO_API_TOKEN;
```

Comment:
In ruby sdk, It hought it was simpler to use the env var FLOW_TOKEN - would suggest we keep consistent. Do you have strong feelings on naming? If so I'll change ruby sdk. If not can we use  FLOW_TOKEN?